docs/portable-modules/portable-modules.md

```diff
@@ -0,0 +1,54 @@
+
+# Portable Modules
+
+It is possible to build a module that targets Windows PowerShell and PowerShell Core today.
+
+One simply needs to target `netstandard1.3` and include an appropriate reference assembly for `System.Management.Automation`, see this [example](https://github.com/Jaykul/NetCoreModuleProof).
+
+Unfortunately, many (most?) non-trivial modules will not work in with Windows PowerShell unless some [facade assemblies](https://github.com/dotnet/standard/blob/6051a7df6d86353f900c46a64f104593647d2904/docs/history/evolution-of-design-time-assemblies.md) are installed.
+
+.Net Core refactored the assemblies so that commonly used types from `mscorlib.dll` are now in `System.Runtime.dll` and less commonly used types are in other assemblies.
+When building against different versions of `netstandard`, an assembly ends up referencing these refactored assemblies instead of `mscorlib`.
+
+A simple module can work on Windows 10 today if it only references the assembly `System.Runtime.dll` because the facade assembly is installed in the GAC on Windows 10.
```

Comment:
Perhaps it makes sense to include a list of `Open Issues` which includes downlevel and facade assemblies which we can research later rather than block this PR